<launch>
  <node name="bag_player" pkg="rosbag" type="play" args="--loop $(find pwc_net)/bag/test_input.bag"/>

  <node name="sample" pkg="pwc_net" type="sample_node" required="true">
    <remap from="image" to="stereo_robot/mobile_base/camera/left/image_raw"/>
    <param name="image_transport" value="compressed"/>
  </node>
</launch>
